package.json

```diff
@@ -74,6 +74,7 @@
     "express": "4.13.3",
     "express-useragent": "1.0.7",
     "filesize": "3.2.1",
+    "find-package-json": "1.0.0",
```

Comment:
I don't love the idea of using a package with so little support. especially for a function that is so few lines!

Would it be better to write our own? Something like:
```js
function findPkgJson(target) {
	let root = path.dirname(target);
	while (path.resolve(root) !== '/') {
		const filepath = path.join(root, 'package.json');
		if (fs.existsSync(filepath)) {
			return JSON.parse(fs.readFileSync(filepath, 'utf8'));
		}
		root = path.join(root, '../');
	}
	throw new Error('could not find a pkg json');
}
```

Reply:
yeah let's just rip it out and find some simpler and more obvious method for finding this information.

in the meantime I'm fine with us 🚢 ing this